src/renderer.rs

```diff
@@ -43,15 +44,14 @@ impl Renderer {
     pub fn new(
         font: &str,
         font_size: impl Into<FontSize>,
-        context: &EGLContext,
-        surface: &EGLSurface,
+        context: &PossiblyCurrentContext,
     ) -> Self {
         unsafe {
             // Setup OpenGL symbol loader.
-            gl::load_with(|symbol| egl::get_proc_address(symbol));
-
-            // Enable the OpenGL context.
-            context.make_current_with_surface(surface).expect("Unable to enable OpenGL context");
```

Comment:
What's the reason for removing that here? Because you made the context current during creation?

Reply:
Yes?